packages/vanilla-router/src/route.ts

```
@@ -0,0 +1,435 @@
+  Register,
+  ResolveFullPath,
+  ResolveId,
+  ResolveParams,
+  RootRouteOptions,
+  RouteConstraints,
+  RouteOptions,
+} from '@tanstack/router-core'
+
+// Router context for route getters - deprecated, router should be passed as parameter
+declare global {
+  var __tanstackRouter: import('@tanstack/router-core').AnyRouter | undefined
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)